gemrb/GUIScripts/InventoryCommon.py

```diff
@@ -620,6 +620,14 @@ def UpdateSlot (pc, slot):
 			Button.SetBAM (SlotType["ResRef"],0,0)
 			Button.SetTooltip (SlotType["Tip"])
 
+		if GameCheck.IsPST():
+			#SlotMap won't be defined if pst/GUIINV is not loaded
```

Comment:
You can just check if SlotMap, it's not true that it won't be defined — you added it to the top of the file.

Reply:
Oh yeah, of course - although I still have to have the indented checks I think, or I'll get an index out of range error from python from trying to read SlotMap[slot]

Reply:
Yes, you do. Just merge the two ifs into one and it'll be even better.